### PR TITLE
Docs & Specs update for Data Flows

### PR DESCRIPTION
**Pull request by azrikahar**
Follow up to #12522, and should be merged together.

- [x] Add flows in docs API reference
- [x] Add oeprations in docs API reference
- [x] Remove webhooks from docs in general
- [x] Add flows in specs
- [x] Add operations in specs
- [x] Remove webhooks from specs

### CODE REVIEW

File: docs/app/settings.md [hunk 1]
```diff
@@ -13,15 +13,14 @@ Only Administrators have access to the Settings Module.
 
 :::
 
-Settings are split into five distinct sections, each described below.
+Settings are split into four distinct sections, each described below.
```
Comment (azrikahar) [inline docs/app/settings.md:16]: do we want to include Translated Strings and Flows? or just Flows, or none (not sure if it's better suit to be included in another PR by Eron)? 🤔

Comment (erondpowell) [reply docs/app/settings.md:16]: Ahh, I do know. This exists as a thing inside the Settings menu.
TBH I'm not sure what it does yet. Translate the app??

Comment (rijkvanzanten) [reply docs/app/settings.md:16]: It's where you manage custom translation strings that you can then use for things like interface options (so you can translate placeholders etc) 🙂 魔法

Comment (erondpowell) [reply docs/app/settings.md:16]: oh. swag!

**Review by rijkvanzanten — APPROVED** [review]
